kit/servo_board.md

```diff
@@ -16,14 +16,16 @@ Board Diagram
 -------------
 <img src="{{ site.baseurl }}/images/content/kit/servo_board_v4_diagram.png" alt="A diagram of a servo board" />
 
-<!--
 Indicators
 ----------
 
-|   LED           | Meaning                | Initial power-up state
-|-----------------|------------------------|----------------------
-| Power           | The board is powered   | On
--->
+| LED           | Meaning                                                                                          | Initial power-up state |
+| ------------- | ------------------------------------------------------------------------------------------------ | ---------------------- |
+| Power         | The board is powered over USB.                                                                   | On                     |
+| 5.5V On       | There is 5.5V power on the board. This usually indicates that the 12V connector rail is powered. | Off                    |
+| Aux On        | There is auxiliary power on the board.                                                           | Off                    |
```

Comment:
What is "Aux On"? This doesn't seem to be included on the diagram. Should we be updating the diagram?

Reply:
Yes the diagram needs updating by the look of things. The LEDs are definitely on the board if you look at one. They are marked on the sticker on the casing as "5.5V On", "Aux On", "Error".